api/v1beta1/osccluster_validation.go

```diff
@@ -101,20 +101,20 @@ func ValidatePort(port int32) (int32, error) {
 }
 
 // ValidateLoadBalancerType check that the  loadBalancerType is a valid
-func ValidateLoadBalancerType(loadBalancerType string) (string, error) {
+func ValidateLoadBalancerType(loadBalancerType string) error {
```

Comment:
I think you can simplify the code using named return values, this removes unnecessary "else" statement.

```suggestion
func ValidateLoadBalancerType(loadBalancerType string) (err error) {
	if loadBalancerType != "internet-facing" && loadBalancerType != "internal" {
		err = errors.New("Invalid LoadBalancerType")
	}

	return
}
```

Reply:
This is a copy past from outscale source repo.
i have not applied any changes to limit diff